### DISCUSSION TITLE
What's new in "PortalName" component

### DISCUSSION THREAD

**Discussion by jonquet** [ontoportal/ontoportal-project#36] [Ideas]
We are trying to revisit the historical component on the Welcome page that was displaying the Latest Notes and Latest Mappigns. 

The idea would be to have a simple UI component on the Welcome page (or somewhere else) called "What's new in PortalName?"
We could see: 
- 3 latest ontologies uploaded to the portal
- 5 latest created concepts/classes
- 3 latest comments or proposals or notes

To get the latest created concepts, we would rely on a new metadata object description properties called "Object creation date property" (we also have Object modification date property) which specified the property used to encode the date of creation of a 
concepts/classes. 

Any ideas / remark about such a component? 

@Bilelkihal Ideas on how we could represent this ?

**Comment by Bilelkihal** [comment]
![image](https://github.com/ontoportal/ontoportal-project/assets/61744974/d3efbbc3-6ec4-48f0-bfad-7668777fb0ed)


@jonquet something similar to this?

**Reply by jonquet** [reply]
Yes. Could be something like this.... and I like the idea of grouipng in "Latest from our community" the latest : notes, reviews, projects, mappigns, proposal

**Comment by syphax-bouazzouni** [comment]
Hello,

In brief 

* We don't have enough activity in those aspects to give valuable data to users. So it will show more the no-usage of these features. 
* Focus more on historical than recent activity metrics. (as did in the AgroPortal landscape https://agroportal.lirmm.fr/landscape)  
* Don't display data (tables, graphs), instead give/infer useful information to users.


In detail 

I'm 50/50 on this; Having and showing analytics to users is a good idea. But the issues will be the same as what we had before, with the block latest_notes 
![image](https://github.com/ontoportal/ontoportal-project/assets/29259906/4b9bd5cf-bf24-43ac-8f81-6f72ebe97e20)

It will show more no-usages of those features (even if we plan to revive and communicate on that feature more in the future). Same for the latest ontologies uploaded, and the latest created concepts/classes, we don't have that much of regular uploads to have valuable graphs or metrics to display to users.

I think we should not focus on "what's new", but more focus on "historical analytics" as did in the landscape, the idea is that as we have a lot of data; we need just to extract useful information from them for people. 

So for example  here is what I propose as historical analytics (the screenshot came from https://agroportal.lirmm.fr/landscape) :
* instead of "3 latest ontologies uploaded to the portal" =>  Most used Ontology (depend on visits + mappings + community notes) 
<img width="613" alt="image" src="https://github.com/ontoportal/ontoportal-project/assets/29259906/d0afe373-c631-4103-a792-3b10e1577631">

* instead of "5 latest created concepts/classes" => Most used Concepts (depending on there mappings + community notes)
* instead of "3 latest comments or proposals or notes" => Most active users (depending on created ontologies + created notes) 
<img width="1142" alt="image" src="https://github.com/ontoportal/ontoportal-project/assets/29259906/389b7516-ac71-4122-bfa0-7cb368815223">

**Comment by syphax-bouazzouni** [comment]
Can you @galviset, @naouelkaram, @jvendetti and @IlariaRosati tell us what do want to see on the homepage of your portal as metrics for your users or institutions?

**Comment by galviset** [comment]
> We don't have enough activity in those aspects to give valuable data to users.

This is our case as well.
However, when we will have more users and resources, we could look forward to have:
- similar to organizations, most active categories
- most recommended concepts/ontologies with the recommender/annotator
- if we get to support SSSOM for mappings, maybe related metrics could be useful. For now, I cannot think of any.

**Comment by jvendetti** [comment]
With regard to the BioPortal homepage, we prefer a clean / minimal design. Adding metrics outside of what we display in the Statistics panel isn't a priority. If we offered the sort of word clouds you depict above, they would likely go on a separate page, as you've done with your Landscape page in AgroPortal.

Also - we've had a lot of back and forth discussions about the notes functionality in Bioportal. It's still something we're not sure we'll keep in the long run. Using notes as part of calculating "most active" may or may not work for us.
